README.md

```diff
@@ -37,11 +37,11 @@ Run `script/server` script to serve the application, or follow these steps:
 2. Start everything with `docker compose up`.
 
 ## Test
-Run `script/test` script to serve the application and run `behave`, or follow these steps:
+Run `script/test` script to serve the application and run `script/cucumber`, or follow these steps:
```

Comment:
The cucumber tests should actually run right when we just run `script/test`?

Reply:
Yes, good catch. I've fixed this in the README